src/lib/Persistence/Legacy/MigrateRichTextNamespaces/Gateway/DoctrineDatabase.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\FieldTypeRichText\Persistence\Legacy\MigrateRichTextNamespaces\Gateway;
+
+use Doctrine\DBAL\Connection;
+use Ibexa\FieldTypeRichText\Persistence\Legacy\MigrateRichTextNamespaces\GatewayInterface;
+
+/**
+ * @internal
+ */
+final class DoctrineDatabase implements GatewayInterface
+{
+    private const CONTENT_ATTRIBUTE_TABLE = 'ezcontentobject_attribute';
+    private const FIELD_TYPE_IDENTIFIER = 'ezrichtext';
+
+    private Connection $connection;
+
+    public function __construct(Connection $connection)
+    {
+        $this->connection = $connection;
+    }
+
+    /**
+     * @param array<string, string> $values
+     *
+     * @throws \Doctrine\DBAL\Exception
+     */
+    public function replaceDataTextAttributeValues(array $values): int
+    {
+        $qb = $this->connection->createQueryBuilder();
+        $qb->update(self::CONTENT_ATTRIBUTE_TABLE);
+
+        $columnName = 'data_text';
+
+        foreach ($values as $oldValue => $newValue) {
```

Comment:
Timing is an essence here, as clients may have millions of rows to migrate. This will cause 6 sqls to be sent to the DB. It might be almost 6 times faster to do it all in on go as in https://github.com/ibexa/fieldtype-richtext/pull/67#discussion_r1120142104 as I assume the replacing is way faster than actually updating the row

Is there actually any reasons for not providing this migration as a .sql script ?